hbw/app/controllers/hbw/tasks_controller.rb

```diff
@@ -46,7 +46,7 @@ def task_id
     end
 
     def form_data
-      params.require(:form_data)
```

Comment:
It's better to submit correct data rather than lowering the strength of checks.

Check forms.js.coffee.